src/fannkuch_redux.rs

```diff
@@ -4,11 +4,35 @@
 // contributed by the Rust Project Developers
 // contributed by TeXitoi
 // contributed by Cristi Cobzarenco (@cristicbz)
+// contributed by Andre Bogus
 
 extern crate rayon;
 
-use std::{cmp, mem};
+use std::cmp;
 use rayon::prelude::*;
+use std::arch::x86_64::*;
```

Comment:
I think you need a `cfg` on this too.

Reply:
Agreed. For the game, it's enough though.